test.js

```diff
@@ -47,14 +47,23 @@ test('zwitch(options)', function (t) {
   t.end()
 })
 
+/**
+ * @param {unknown} value
+ */
 function invalid(value) {
   throw new Error('Invalid: `' + value + '`')
 }
 
+/**
+ * @param {{ [key: string]: unknown, type: string }} value
```

Comment:
```suggestion
 * @param {{[key: string]: unknown, type: string}} value
```

Btw, no tooling is checking/catching this, but I prefer whitespace in those braces in a way that resembles how objects are formatted in JS too